tools/aruco_checker.py

```diff
@@ -102,150 +97,133 @@ def detecting_markers_location(self, image):
             y = int(y)
```

Comment:
line 96 change to `x,y,z = int(marker[0][0]), int(marker[0][1]), int(marker[0][2])`